asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py

```diff
@@ -189,20 +201,20 @@ def check_inputs(cls, values):
                 "Must specify postera_molset_name if uploading to postera."
             )
 
-        if fragalysis_dir and structure_dir:
-            raise ValueError("Cannot specify both fragalysis_dir and structure_dir.")
-
-        if not fragalysis_dir and not structure_dir:
-            raise ValueError("Must specify either fragalysis_dir or structure_dir.")
+        # fragalysis dir, structure dir and PDB file are mutually exclusive
+        if not fragalysis_dir and not structure_dir and not pdb_file:
+            raise ValueError(
```

Comment:
it looks like you got rid of the logic to fail if more than one is specified?

Reply:
Good spotting.